provider/datagen/src/transform/icuexport/collator/test.rs

```diff
@@ -39,7 +15,8 @@ struct TestCase<'a> {
     expectation: Ordering,
 }
 
-fn check_expectations(collator: &Collator, cases: &[TestCase<'_>]) {
+fn check_expectations(locale: &Locale, options: CollatorOptions, cases: &[TestCase<'_>]) {
+    let collator = Collator::try_new(DatagenProvider::for_test(), locale.clone(), options).unwrap();
```

Comment:
Nit: `DatagenProvider::for_test()` re-creates the caches each time. I had been using the `lazy_static` in order to re-use the caches as much as possible.

Reply:
It does not. It uses a single static provider for all tests, which is an improvement over only sharing it between Collator tests

Reply:
Oh cool, you put the lazy_static inside of `DatagenProvider::for_test`. Sorry for the noise.